test/DistanceStraightTest.cpp

```diff
@@ -0,0 +1,293 @@
+/**
+ *  @file   DistanceStraightTest.cpp
+ *  @brief  DistanceStraightクラスのテスト
+ *  @author Negimarge
+ */
+
+#include "DistanceStraight.h"
+#include <gtest/gtest.h>
+#include "Controller.h"
+#include "Measurer.h"
```

Comment:
```suggestion
#include "Measurer.h"
#include "Controller.h"
```
テストの方は影響あるかわからんけど、Measurer.hできるだけ先にincludeしてほしい